app/models/katello/glue/candlepin/pool.rb

```diff
@@ -81,20 +81,28 @@ def backend_data
         self.class.candlepin_data(self.cp_id)
       end
 
+      def stacking_subscription(org_label, stacking_id)
+        subscription = ::Katello::Subscription.find_by(:product_id => stacking_id)
```

Comment:
You are going to need to return this subscription if it is found, I wound up getting null subscriptions

here is the diff I used that fixed it.
```
diff --git a/app/models/katello/glue/candlepin/pool.rb b/app/models/katello/glue/candlepin/pool.rb
index 1bfda4d..2110794 100644
--- a/app/models/katello/glue/candlepin/pool.rb
+++ b/app/models/katello/glue/candlepin/pool.rb
@@ -85,8 +85,9 @@ module Katello
         subscription = ::Katello::Subscription.find_by(:product_id => stacking_id)
         if subscription.nil?
           found_product = ::Katello::Resources::Candlepin::Product.find_for_stacking_id(org_label, stacking_id)
-          ::Katello::Subscription.find_by(:product_id => found_product['id']) if found_product
+          subscription = ::Katello::Subscription.find_by(:product_id => found_product['id']) if found_product
         end
+        subscription
       end
 
       def import_data
```